packages/chart/src/components/LinearChart.jsx

```diff
@@ -91,6 +100,18 @@ export default function LinearChart() {
       }
     }
 
+    // DEV-3263 - If Confidence Intervals in data, then need to account for increased height in max for YScale
```

Comment:
Personally adding the ticket numbers in feels confusing. This is 3163 but the comment references 3263. It could be a spelling error or it could be two different tickets. I still feel like ticket numbers can be left to git.